crates/cac_client/src/lib.rs

```diff
@@ -102,7 +112,24 @@ impl Client {
             return Err("Invalid tenant".to_string());
         }
         let config = resp.json::<Config>().await.map_err_to_string()?;
-
+        let config_cache = Cache::builder()
+            .weigher(|_key, value: &Map<String, Value>| -> u32 {
+                json!(value.to_owned())
```

Comment:
lets avoid macros when it is known that the data is already of `serde_json::Value` compatible type
```suggestion
                Value::Object(value.to_owned())
```